### ISSUE TITLE
Is Proton VPN Ever Going to be Accessible with Screen Readers?

### ISSUE THREAD

**Issue by centrum-permanebit** [ProtonVPN/win-app#95]
>  We are happy to answer your questions about the code or discuss technical ideas. 
 
Please complete the following checklist (by adding [x]):
- [ ] I have searched open and closed issues for duplicates x
- [ ] This isn't a feature request x
- [ ] This is not a report about my app not working as expected x

----------------------------------------
The Windows app for Proton VPN is nearly unusable with screen readers like NVDA. Buttons don't have labels, UI elements are not labeled, and the settings page is confusing to navigate using a keyboard.
Even after reporting the bug to the customer support multiple times, nothing has been done so far.
This leads to  questions  like is the  app ever going to be accessible with screen readers? do the developers not care about accessibility? and are screen reader/visually impaired users not valid customers of Proton?
